packages/charts-utils/src/constants.ts

```diff
@@ -1,2 +1,10 @@
 export const TO_RADIANS = Math.PI / 180;
 export const TO_DEGREES = 180 / Math.PI;
+export const INTERVAL_UNITS: { [key: string]: string } = {
```

Comment:
in charts-utils we have `const INTERVAL_TO_PRECISION: Record<string, TimePrecision>`

please unify